packages/astro/src/core/app/index.ts

```diff
@@ -97,10 +97,12 @@ export class App {
 		}
 
 		const response = await render({
+			adapterName: manifest.adapterName,
```

Comment:
TODO use this to provide a better error message for an adapter that does not support it.